### PR TITLE
U ibranch

### PR DESCRIPTION
**Pull request by IndignantToaster**
Super simple stuff...maybe only things to address would be spacing and some "questionable names".  Evn the CSS is placeholder just to make it easier to test with, so that's all stuff that will prolly be changed.  I need to understand what and how this app is gonna work more if we want to make a UI tho............

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+body {
```
Comment (eriwang) [inline stylesheet.css]: Let's move this entire file to the `static/` directory

Comment (eriwang) [inline stylesheet.css]: I personally prefer using `<div id="content"...`, I think it gives a little more control for future things we might add, though right now it's functionally the same

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+#uploadButton {
+    height: 20px;
+    padding: 20px;
+    margin: 0 auto;
+}
+
+#workspace {
+    display: flex;
```
Comment (eriwang) [inline stylesheet.css]: Is the flex css necessary for an empty div?

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+body {
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+}
+
+#mainWindow {
```
Comment (eriwang) [inline stylesheet.css]: I personally prefer `main-window`, which i think is the widely used convention (it looks a bit more consistent with HTML and CSS code if you ask me, where capital letters are rare), but your call

**Review by eriwang — CHANGES_REQUESTED** [review]

File: stylesheet.css (hunk 3 above)
Comment (IndignantToaster) [reply stylesheet.css]: camel case is love, camel case is life.  but fine.

File: stylesheet.css (hunk 2 above)
Comment (IndignantToaster) [reply stylesheet.css]: nop, guess not

File: stylesheet.css (hunk 1 above)
Comment (IndignantToaster) [reply stylesheet.css]: consider it done.

Comment (IndignantToaster) [reply stylesheet.css]: ooooooooooooooooooook

File: static/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+#main-window {
+    display: flex;
+    align-items: center;
+    width: 300px;
+    height: 100px;
+    background-color: deepskyblue;
+    margin: 0;
+}
+
+#uploadButton {
```
Comment (eriwang) [inline static/stylesheet.css]: You missed one

File: static/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
+body {
```
Comment (eriwang) [inline static/stylesheet.css:1]: What does having a body with this as well as a content div that has the same thing do??

**Review by eriwang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**